<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgResumoCnis</class>
 <widget class="QWidget" name="wdgResumoCnis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#wdgCadastroCliente {
	background-color: white;
}

#frCabecalho, #frMiolo, #frRodape {
	background-color: white;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frPrincipal">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frCabecalho">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">/*------------------------------------------- Frame -------------------------------------------*/
#frFirulaEmpresa, #frFirulaContrib,
#frFirulaBeneficios {
	background-color: #3F4E8C;
	border: 0px solid none;
	border-radius: 2px;
}

#frFirulaContrib,
#frFirulaBeneficios {
	background-color: white;
	border: 0px solid none;
	border-radius: 2px;
}


/*------------------------------------------- Push Button -------------------------------------------*/
#pbVoltar {
	background-image: url(:/cabecalho/back.png);
	background-position: center;
	background-repeat: no-repeat;

	background-color: #F4F5F8;
	border-radius: 8px;
}

#pbVoltar:hover {
	background-image: url(:/voltar/back.png);
	background-position: center;
	background-repeat: no-repeat;

	background-color: rgb(228, 233, 248);
	border-radius: 8px;
}

#pbEmpresas {
	font: 12pt &quot;Avenir LT Std&quot;;
	color: #3F4E8C;
	font-weight: 750;

	background-color: transparent;
	border: 0px solid none;

	margin: 4px;
}

#pbBeneficios,
#pbContrib {
	font: 12pt &quot;Avenir LT Std&quot;;
	color: black;

	background-color: transparent;
	border: 0px solid none;

	margin: 4px;
}

/*------------------------------------------- Label -------------------------------------------*/
#lbInfoTitulo {
	font: 16pt &quot;Avenir LT Std&quot;;
	color: #3F4E8C;
	font-weight: 750;
}

#lbNomeCliente {
	font: 12pt &quot;Avenir LT Std&quot;;
	color: #3F4E8C;
	font-weight: 750;

	padding: 4px;
}

#lbCpfCliente {
	font: 11pt &quot;Avenir LT Std&quot;;
	color: #3F4E8C;

	padding: 4px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frTitulo">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pbVoltar">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignVCenter">
             <widget class="QLabel" name="lbInfoTitulo">
              <property name="text">
               <string>Resumo do CNIS</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frInfoPessoal">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="lbNomeCliente">
                 <property name="text">
                  <string>nomeCliente</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbCpfCliente">
                 <property name="text">
                  <string>cpfCliente</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>883</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QFrame" name="frBotoes">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QFrame" name="frEmpresas">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item alignment="Qt::AlignBottom">
                   <widget class="QPushButton" name="pbEmpresas">
                    <property name="text">
                     <string>Empresas</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                    <property name="checked">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item alignment="Qt::AlignTop">
                   <widget class="QFrame" name="frFirulaEmpresa">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>4</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>4</height>
                     </size>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frContrib">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item alignment="Qt::AlignBottom">
                   <widget class="QPushButton" name="pbContrib">
                    <property name="text">
                     <string>Contribuições</string>
                    </property>
                   </widget>
                  </item>
                  <item alignment="Qt::AlignTop">
                   <widget class="QFrame" name="frFirulaContrib">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>4</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>4</height>
                     </size>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frBeneficios">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item alignment="Qt::AlignBottom">
                   <widget class="QPushButton" name="pbBeneficios">
                    <property name="text">
                     <string>Benefícios</string>
                    </property>
                   </widget>
                  </item>
                  <item alignment="Qt::AlignTop">
                   <widget class="QFrame" name="frFirulaBeneficios">
                    <property name="minimumSize">
                     <size>
                      <width>0</width>
                      <height>4</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>4</height>
                     </size>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frMiolo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/*-------------------------------------------  Stack --------------------------------------------*/
#stkCliente {
	background-color: white;
}

/*-------------------------------------------  Widget --------------------------------------------*/
#pgEmpresas, #pgContribuicoes, 
#pgBeneficios {
	background-color: white;
}

/*-------------------------------------------  Label --------------------------------------------*/


/*-------------------------------------------  Frame --------------------------------------------*/


/*---------------------------- Push Button ----------------------------------*/
#pbInserir {
	font: 12pt &quot;Avenir LT Std&quot;;
	color: #3F4E8C;
	
	border: 2px solid #3F4E8C;
	border-radius: 8px;
	padding: 6px;
}

/*----------------------------  Line Edit ---------------------------------------*/


/*----------------------- Scroll Area  and  QWidget ------------------------*/
#scaPrincipal, #wdgScroll {
	background-color: white;
}

/*------------------------------ Check Box -------------------------------------*/


/*----------------------------- Combo box --------------------------------------*/


/*----------------------------- Date Edit --------------------------------------*/
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QStackedWidget" name="stkCliente">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="pgEmpresas">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="horizontalSpacing">
              <number>32</number>
             </property>
             <property name="verticalSpacing">
              <number>16</number>
             </property>
             <item row="0" column="0">
              <widget class="QFrame" name="frame">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                 <widget class="QFrame" name="frame_2">
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <spacer name="horizontalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>479</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pbInserir">
                     <property name="minimumSize">
                      <size>
                       <width>80</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Inserir</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QScrollArea" name="scaPrincipal">
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="wdgScroll">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>586</width>
                     <height>406</height>
                    </rect>
                   </property>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pgContribuicoes">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="horizontalSpacing">
              <number>32</number>
             </property>
             <property name="verticalSpacing">
              <number>16</number>
             </property>
            </layout>
           </widget>
           <widget class="QWidget" name="pgBeneficios">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="horizontalSpacing">
              <number>32</number>
             </property>
             <property name="verticalSpacing">
              <number>16</number>
             </property>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frRodape">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">/*----------------------------------------- Push Button---------------------------------------------*/
#pbSalvarDados {
	
	font: 12pt &quot;Avenir LT Std&quot;;
	color: white;

	background-color: #3F4E8C;
	border-radius: 8px;

	padding: 16px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>16</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Resources/wdgResumoCNIS.qrc"/>
 </resources>
 <connections/>
</ui>
